### DISCUSSION TITLE
Running Latest Beta and Getting Constant Error Popup

### DISCUSSION THREAD

**Discussion by dangerzone922** [EvanMulawski/FanControl.CorsairLink#132] [Q&A] [answered]
![image](https://github.com/EvanMulawski/FanControl.CorsairLink/assets/22600272/9f9ee955-9433-493f-8609-180e216d2bd6)

This is what I keep getting. Running Commander XT with all 6 fan slots occupied with LL!20's.

I looked around Fan Control and even the source code of the plugin and DID find the env variable it wants me to set to 1, but only mentioned as a parameter to load it and not the source.

I also notice that running this disables the RGB through Hardware Setting that is used when iCue is not running. I have to briefly start up iCue after Fan Control just to get RGB working and then kill off iCue to make the Hardware Lighting RGB work again.

I will say that other than these issues, this plugin controls my fans perfectly!

**Comment by EvanMulawski** [comment, marked as answer]
Hi @dangerzone922, this plugin will not work properly if iCUE is running or the device is in hardware mode (for hardware lighting). The plugin places the device in software mode because that is the only way for the fans to be controlled by Fan Control. For the Commander CORE XT, you will need to use SignalRGB to control lighting in software mode while Fan Control is running.

**Reply by dangerzone922** [reply]
Thanks for getting back to me so fast! SignalRGB is pretty nice it seems. All in one place. Errors gone and everything works great!
